webapp/sass/profile.scss

```diff
@@ -223,6 +224,13 @@ $pane-width: 6px;
     margin: 0 $pane-width;
     width: 50%;
 
+    .flamegraph-header {
```

Comment:
Can you add padding bottom to this. I think `padding-bottom: 5px` probably would work here.
![image](https://user-images.githubusercontent.com/23323466/112919855-b7fd1500-90bc-11eb-8dd6-c67f4580e2a2.png)